deploy-manage/monitor/logging-configuration/logfile-audit-output.md

```diff
@@ -8,17 +9,19 @@ applies:
   stack: all
 ---
 
-% evaluate the applies section
+# Elasticsearch logfile audit output [audit-log-output]
 
-# Logfile audit output [audit-log-output]
+The `logfile` audit output is the only output for auditing. By default, it writes data to the `<clustername>_audit.json` file in the logs directory. The file is also rotated and archived daily or upon reaching the 1GB file size limit.
 
-The `logfile` audit output is the only output for auditing. It writes data to the `<clustername>_audit.json` file in the logs directory.
+In self-managed clusters, you can configure how the `logfile` is written in the `log4j2.properties` file located in `ES_PATH_CONF` (or check out the relevant portion of the [log4j2.properties in the sources](https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/core/src/main/config/log4j2.properties)). However, **Elastic strongly recommends using the default Log4j2 configuration**.
+
+Orchestrated deployments (Elastic Cloud, Elastic Cloud Enterprise (ECE), and Elastic Cloud on Kubernetes (ECK)) do not support changes in `log4j2.properties` files of the {{es}} instances.
 
 ::::{note} 
 If you overwrite the `log4j2.properties` and do not specify appenders for any of the audit trails, audit events are forwarded to the root appender, which by default points to the `elasticsearch.log` file.
 ::::
 
-
+For {{es}} configuration options that control event filtering in audit logs, refer to [](./configuring-audit-logs.md).
 
 ## Log entry format [audit-log-entry-format] 
```

Comment:
is the json file accessible, or do we think it's ok even if it's not because the monitoring instance consumes the json regardless of deployment type

Reply:
I think it's ok because even if the user doesn't see `file` in orchestrated deployments, they will end up seeing the `json`. So this informative section is the most important part of this file for all deployment types.